java/java.hints/src/org/netbeans/modules/java/hints/suggestions/Tiny.java

```diff
@@ -400,4 +402,83 @@ protected void performRewrite(final TransformationContext ctx) {
 
     }
 
+    @Hint(displayName = "#DN_org.netbeans.modules.java.hints.suggestions.Tiny.inlineRedundantVar", description = "#DESC_org.netbeans.modules.java.hints.suggestions.Tiny.inlineRedundantVar", category="suggestions", hintKind=Kind.ACTION, severity=Severity.HINT)
```

Comment:
nitpick: I believe it would improve readability (esp for future diffs) if this wouldn't be a single line, but this is consistent with the rest of the file so its not really a complaint - just my opinion

Reply:
Let it consistent with the rest of the file.